packages/data/.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     // typescript
     '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
     '@typescript-eslint/explicit-function-return-type': 'off',
+    '@typescript-eslint/ban-ts-ignore': 'off',
```

Comment:
This rule has been deprecated in favour of the ban-ts-comment rule. It will be removed in a future version of this plugin.

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/ban-ts-ignore.md

Reply:
Good catch, gonna leave it as is for now since it seems like the new option isn't working well with vscode